receiver/prometheusexec/README.md

```diff
@@ -0,0 +1,126 @@
+# This receiver is currently under development, do not use.
+// TODO: Remove above message when done
+
+# Prometheus_exec Receiver
+
+This receiver makes it easy for a user to collect metrics with the Collector from third-party services such as MySQL, Apache, Redis, etc. It's meant for people who want a plug-and-play solution to getting metrics from those third-party services that don't natively export metrics or speak any instrumentation protocols. Through the configuration file, you can indicate which binaries to run (usually [Prometheus exporters](https://prometheus.io/docs/instrumenting/exporters/), which are custom binaries that will expose the services' metrics to the Prometheus protocol). It supports starting binaries with flags and environment variables, retrying them with exponentional backoff if they crash, string templating, and random port assignments. If you do not need to spawn the binaries locally, please condider using the [core Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/prometheusreceiver) or the [Simple Prometheus receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/master/receiver/simpleprometheusreceiver).
+
+## Overview
+Here's an example Collector configuration file that starts two `prometheus_exec` receivers and two exporters.
+
+```yaml
+receivers:
+    prometheus_exec/mysqld:
+        exec: ./mysqld_exporter --port {{port}}
+        scrape_interval: 10s
+        port: 9104
+        env:
+          - name: DATA_SOURCE_NAME
+            value: user:password@(url:port)/
+
+    prometheus_exec/postgresql:
+        exec: ./postgres_exporter
+        env:
+          - name: DATA_SOURCE_NAME
+            value: postgresql://user:password@url:port/
+
+exporters:
+    stackdriver:
+        project: "project_name"
+    file:
+        path: ./file_name.json
+
+service:
+    pipelines:
+        metrics:
+            receivers: [prometheus_exec/mysqld, prometheus_exec/postgresql]
+            exporters: [stackdriver, file]
+```
+
+## Config
+For every `prometheus_exec` defined in the configuration file, a command will be run (you *should* start a binary for this receiver to serve its purpose) and an equivalent Prometheus receiver will be instantiated to scrape its metrics.
```

Comment:
```suggestion
For each `prometheus_exec` defined in the configuration file, the specified command will be run. This command *should* start a binary that exposes Prometheus metrics on the specified port. A Prometheus receiver will be instantiated to scrape this endpoint.
```

Reply:
Thanks for the suggestions. I kept most of what you wrote, but removed the part about ports and such since there's multiple ways for the user to go about choosing a port!